hyperspace/metrics/README.md

```diff
@@ -0,0 +1,45 @@
+## Hyperspace Metrics
+
+This crate provides a prometheus server for collecting useful information about the relayer's operation.
+
+### Server
+
+The server can be spawned by calling `init_prometheus` with the server address and a prometheus registry.  
+Metrics can be requested on the `/metrics` route via an http get request.
+
+**Setting up the prometheus server**  
+
+```rust 
+    let registry = Registry::new_custom(Some("hyperspace".to_string()), None).expect("this can only fail if the prefix is empty");
+    let metrics_a = Metrics::register(any_chain_a.name(), &registry)?;
+    let metrics_b = Metrics::register(any_chain_b.name(), &registry)?;
+    let mut metrics_handler_a = MetricsHandler::new(registry.clone(), metrics_a);
+    let mut metrics_handler_b = MetricsHandler::new(registry.clone(), metrics_b);
+    metrics_handler_a.link_with_counterparty(&mut metrics_handler_b);
+    let addr = "127.0.0.1:8080".parse()?;
+    tokio::spawn(init_prometheus(addr, registry.clone()));
+```
```

Comment:
don't we already do this internally?

Reply:
Yeah, in the hyperspace CLI, but just added it, for someone looking at the metrics crate standalone.